plugins/json-schema/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# @flatfile/plugin-json-schema
+# @flatfile/plugin-convert-json-schema
```

Comment:
Was this intentional? Do we need to adjust/fallback names for `npm`?

Reply:
Yes, we renamed the plugin when it was released, but the changelog never got updated.